Improve FlashManager (#90)
sonata-project/twig-extensions: docs/reference/flash_messages.rst

.. code-block:: jinja

    {% include '@SonataTwig/FlashMessage/render.html.twig' %}

> You can also use your own template, some example you can find below:

Reviewer: ```suggestion
You can also use your own template. Below, you can see an example:
```
